flow2ml/AutoResults.py

```diff
@@ -64,7 +64,7 @@ def precision_recall_curve(self,figure_name="prc.jpeg"):
         plt.title('Precision Recall Curve', size = 35)
         plt.savefig(self.results_path+'/'+figure_name)
 
-    def get_results_pdf(self,file_name="report"):
+    def get_results_docx(self,file_name="report"):
```

Comment:
Will this docx display uniformly for any version of Windows? Because structure of docx changes according to Windows version. So please try to create pdf to store all images of results.

Reply:
Yes this docx will display uniformly for any version of windows. My later PR would involve converting this docx to pdf as there is no efficient way to do so yet.